instanovo/diffusion/multinomial_diffusion.py

```diff
@@ -321,9 +321,26 @@ def forward(self, x_0: torch.LongTensor, **kwargs: dict) -> torch.FloatTensor:
         t = torch.randint(0, self.time_steps - 1, (x_0.shape[0],)).to(x_0.device)
 
         # 2. Compute L_t
+        loss = self._compute_loss(t=t, x_0=x_0, **kwargs).mean()
+
+        # 4. Calculate prior KL term
```

Comment:
Numbering is mixed up

```suggestion
        # 3. Calculate prior KL term
```

Reply:
Fixed in https://github.com/instadeepai/InstaNovo/pull/14/commits/464d3811d557eb5abcc033ea96f671a83f720599